server/UserRegistration.java

```diff
@@ -0,0 +1,357 @@
+package newbank.server;
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.Scanner;
+
+public class UserRegistration {
+
+    private static Scanner sc = new Scanner(System.in);
+        UserRegistration ur = UserRegistration.newUser();
```

Comment:
What is this used for? I think this is not needed, as you have the call to `newUser` already in the other file.